Yank/experiment.py

```diff
@@ -353,6 +359,7 @@ class ExperimentBuilder(object):
         'platform': 'fastest',
         'precision': 'auto',
         'switch_experiment_interval': 0,
+        'nodes_per_experiment': None
```

Comment:
Ah, right! I'll call it `processes_per_experiment` and add it to the website docs.